shell/platform/common/BUILD.gn

```diff
@@ -91,6 +91,9 @@ source_set("common_cpp_accessibility") {
   public_configs =
       [ "//flutter/third_party/accessibility:accessibility_config" ]
 
+  configs -= [ "//build/config/compiler:no_rtti" ]
+  configs += [ "//build/config/compiler:rtti" ]
```

Comment:
enable_nui_support variable information set in shell/platform/tizen/BUILD.gn cannot be imported into this context. 
If you know how to pass nui support info to this file and the gn file of accessibility, please let me know.